docs/nbis_annotation/schedule.md

```diff
@@ -14,22 +14,23 @@ Genome Annotation Service
 <hr>
 **Thursday 1st**
 
-**09.00-09.30:** [Lecture: Structural annotation programs and pipelines](https://github.com/SGBC/course/blob/master/docs/nbis_annotation/slides/Functional_annotation.pptx?raw=true)   
-**10.00-10.15:** Coffee break    
-**10.15-11.30:** [Practical 1: Assembly assessment + Abinitio annotation](practical_session/practical1.md)   
-**11.30-12.30:** Lunch   
-**12.30-13.00:** [Lecture: Structural annotation programs and pipelines (part 2)](https://github.com/SGBC/course/blob/master/docs/nbis_annotation/slides/Functional_annotation.pptx?raw=true)  
-**13.00-16.00:** [Practical 2: Structural annotation of eukaryote genomes (incl. coffee break)](practical_session/practical2.md)<br/>
+**09.00-9.30** [Lecture: Structural annotation programs and pipelines](slides/Structural_annotation.pptx)<br/> 
+**10.00-10.15** Coffee break  <br/> 
+**10.15-11.30** [Practical 1: Assembly assessment + Abinitio annotation](practical_session/practical1.md) <br/> 
+**11.30-12.30** Lunch  <br/> 
+**12.30-13.00**	[Lecture: Structural annotation programs and pipelines (part 2)](slides/Structural_annotation.pptx)<br/>
+**13.00-16.00** [Practical 2: Structural annotation of eukaryote genomes (incl. coffee break)](practical_session/practical2.md)<br/> 
+
 
 <hr>
 **Friday 2nd**
 
-**09.00-09.15:** Summary of yesterday’s exercise  
-**09.15-09.30:**  Lecture: Introduction to manual curation  
-**10.00-12.00:** [Practical 3: Manual curation(incl. coffee break)](practical_session/practical3_manualCuration.md)  
-**11.30-12.30:** Lunch   
-**12.30-13.00:** [Lecture: Functional annotation](https://github.com/SGBC/course/blob/master/docs/nbis_annotation/slides/Structural_annotation.pptx)    
-**13.00-15.30:** [Practical 4: Functional annotation (incl. coffee break)](practical_session/practical4_funcAnnotInterp.md)  
-**15.30-16.00:** Wrap-up   
+**09.00-9.15** Summary of yesterday’s exercise  
+**09.15-09.30** Lecture: Introduction to manual curation  
+**10.00-12.00** [Practical 3: Manual curation(incl. coffee break)](practical_session/ExerciseManCuration)  
+**11.30-12.30** Lunch   
+**12.30-13.00** [Lecture: Functional annotation](slides/Functional_annotation.pptx)  
+**13.00-15.30** [Practical 4: Functional annotation (incl. coffee break)](practical_session/ExerciseFuncAnnotInterp)  
```

Comment:
breaks the link to

Reply:
your tutorials are **not** named `ExerciseManCuration` and `ExerciseFuncAnnotInterp` but rather practical3_manualCuration.md` and `practical4_funcAnnotInterp.md`, respectively. Rename your files or change the schedule